lib/options.dart

```diff
@@ -4,39 +4,65 @@ import 'dart:ui';
 import 'package:image/image.dart' hide Color;
 import 'package:iproov_flutter/enums.dart';
 
+// TODO: For all the classes in this file, identify which properties need to be null
+// and which ones don't. Non-nullable should be the default.
+// Note: non-nullable variables need to be required or have a default value in the constructor.
```

Comment:
Left this as a TODO so you can decide which properties should become non-nullable.